pkg/source/vmware/client.go

```diff
@@ -208,7 +211,7 @@ func (c *Client) ExportVirtualMachine(vm *migration.VirtualMachineImport) (err e
 			vm.Status.DiskImportStatus = append(vm.Status.DiskImportStatus, migration.DiskInfo{
 				Name:     i.Path,
 				DiskSize: i.Size,
-				BusType:  adapterType(i.DeviceId),
+				BusType:  busType,
```

Comment:
we are ignoring the bus override for vmware imports. should we check and apply the override here?

Reply:
The `defaultDiskBusType` field is used to specify the "default" or fallback type if auto-detection does not work. OpenStack does not have an auto-detection, so this is THE bus type for this source. VMware instead supports auto-detection, so the `defaultDiskBusType` is ONLY used ass fallback if the auto-detection fails.

Reply:
sorry my bad. the vm spec bustype is only applied as default in the bus detection logic